### DISCUSSION TITLE
Chromiun? Chrome Extensions?

### DISCUSSION THREAD

**Discussion by cfuendev** [minbrowser/min#1794] [Q&A] [answered]
Min's based on Chromiun, right? I'm new to Min and a few hours ago I checked my linked devices after linking my phone to Whatsapp Web on Min, and I noticed the fact that the Min session appeared as "Chrome" on the list. And, if Min is based on Chromiun, are Chrome extensions supported? Or was that part removed to endorse the usage of userscripts?

I wanted to port some of the extensions I had on Opera to Min as userscripts. But I was wondering if that was necessary or if I could just slap them onto Min from the Chrome store. I'm feeling too lazy to open my google account and try it out by myself and I wanted to get a clear response on if the browser is Chromiun-based, so I thought it made sense to ask here.

**Comment by PalmerAL** [comment, marked as answer]
Electron doesn't support Chromium extensions very well (there's some minimal support, but not enough to make most extensions compatible) - mainly because Electron only uses the Chromium rendering engine, and not the UI and other components from Chromium. See #1.

Userscripts only support a subset of the things extensions do, but full extension support would be a huge time investment that hasn't been feasible.

**Comment by marcusquinn** [comment]
WebCatalog App got Chrome Extensions working experimentally with Electron for interest:
* https://github.com/webcatalog/webcatalog-app

**Reply by PalmerAL** [reply]
Oh this is really interesting. It looks like they use this library: https://www.npmjs.com/package/electron-chrome-extensions

We should definitely investigate that.
